build/creator/ZcToken.sol

```diff
@@ -31,7 +31,8 @@ contract ZcToken is Erc20, IERC5095 {
         underlying = _underlying;
         maturity = _maturity;
         cToken = _cToken;
-        redeemer = IRedeemer(_redeemer);
+        // TODO historically we just store the address and hydrate via interface. should we break that pattern here?
+        redeemer = _redeemer;
```

Comment:
Unless there's a functional reason not to, I don't think we should. I did notice for some getters in solidity, it does become necessary to store the attribute as its type as hydrating an interface doesn't play well with the ABI.